### PR TITLE
[BEAM-7794] Ignore blocking DynamoDBIOtest class

### PR DESCRIPTION
**Pull request by iemejia**
R: @jbonofre

### CODE REVIEW

File: sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/dynamodb/DynamoDBIOTest.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
 import org.mockito.Mockito;
 
 /** Test Coverage for the IO. */
+@Ignore
```
Comment (jbonofre) [inline sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/dynamodb/DynamoDBIOTest.java]: Maybe worth to add comment of the reason for the ignore, with something like `@Ignore("reason is ...")`

**Comment by jbonofre** [comment]
I'm waiting for Jenkins before merging.

**Comment by iemejia** [comment]
Run JavaPortabilityApi PreCommit

File: sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/dynamodb/DynamoDBIOTest.java (hunk 1 above)
Comment (iemejia) [reply sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/dynamodb/DynamoDBIOTest.java]: Good point, updated now.

**Comment by iemejia** [comment]
Run JavaPortabilityApi PreCommit

**Comment by iemejia** [comment]
Error is unrelated to this PR, I am merging eagerly because I need to cherry pick it in the 2.14 branch. Thanks JB !

**Comment by jbonofre** [comment]
+1, thanks !

**Pull request merged**